idea-plugin/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java'
+apply plugin: 'com.palantir.external-publish-intellij'
+
+intellij {
+    pluginName = "gradle-consistent-versions"
+    updateSinceUntilBuild = true
+    version = "2024.1"
+    plugins = ['java', 'org.jetbrains.plugins.gradle']
+}
+
+patchPluginXml {
+    pluginDescription = "Language plugin for version.props allowing for auto-lock updating"
+    version = "1.0.0" // For some reason project.version is unspeificed and thus this fails
+    sinceBuild = '241' // TODO: test against this version of IntelliJ to ensure no regressions
+    untilBuild = ''
+}
+
+afterEvaluate {
+    tasks.named('test').configure { it.javaLauncher = null }
+}
```

Comment:
Should move this to the external-publish-plugin.

Reply:
done see https://github.com/palantir/gradle-external-publish-plugin/pull/460